src/fides/api/ops/graph/config.py

```diff
@@ -416,6 +416,7 @@ class Collection(BaseModel):
     fields: List[Field]
     # an optional list of collections that this collection must run after
     after: Set[CollectionAddress] = set()
+    erase_after: Set[CollectionAddress] = set()
```

Comment:
To clarify, one of the reasons you added a separate "erase_after" param from "after", was not just that this wasn't getting respected in the erasure section, but that the "erase_after" ordering may conflict with "after" ordering?
So it needs to be a legitimately separate param.

Reply:
Right that was the main reason, the dataset references and after field work well together but I saw instances where this order conflicted with the erasure order and I would get incomplete traversals.